index.html

```diff
@@ -2403,14 +2401,17 @@
       </p>
     </section>
     <section> <h3>URL patterns</h3>
-        A <dfn>URL pattern</dfn> is a valid <a>URL</a> that can be used to match
-        <a>Web NFC Id</a> of <a>Web NFC message</a>.
+        A <dfn>URL pattern</dfn> is a <a>URL</a> that can be used to match
+        <a>Web NFC Id</a> of a <a>Web NFC message</a>.
+        A <dfn>valid URL pattern</dfn> is a valid <a>URL</a> whose
+        <a href="#dfn-url-scheme">scheme</a> component is equal to
+        <code>"https"</code>.
```

Comment:
I will change to URL => 'URL record' to avoid ambiguity. All URL parts are lowercased by default.
https://url.spec.whatwg.org/#scheme-start-state.